cutter/cutter/main.py

```diff
@@ -20,6 +20,29 @@
 
 from .youtube import Youtube
 
+videos_cut = prom.Counter(
+	'videos_cut',
+	'Number of videos successfully cut',
+	['video_channel', 'video_quality']
+)
+
+videos_uploaded  = prom.Counter(
+	'videos_uploaded',
+	'Number of videos successfully uploaded',
+	['video_channel', 'video_quality', 'upload_location']
+)
```

Comment:
what's the difference between a video being cut and a video being uploaded? we do both at the same time.

Reply:
reading further, it looks like the difference is in finalize beginning and finalize ending. maybe update the names to reflect that? say, `videos_finalizing` and `videos_uploaded`

Reply:
I think I will switch to just one counter for videos uploaded